src/pages/send/send.spec.ts

```diff
@@ -605,10 +610,15 @@ describe('SendPage', () => {
         WalletTabsProvider
       );
       const events: Events = testBed.get(Events);
-      const navParams: NavParams = testBed.get(NavParams);
+      instance.navParams = {
+        data: {
+          amount: '1.00000',
+          coin: Coin.BCH
+        }
+      };
       const amount = '1.00000';
       const coin = Coin.BCH;
-      spyOn(navParams, 'get').and.returnValues(amount, coin);
+      spyOn(instance.navParams, 'data').and.returnValues(amount, coin);
```

Comment:
I think this line is not necessary. You could remove it

Reply:
+1 Thanks!